### ISSUE TITLE
Preloading breaks animations.

### ISSUE THREAD

**Issue by AtomicTEM** [mastercoms/tf2cfg#176]
**Describe the bug**
When I use map background to preload animations mods to bypass sv pure, animations are broken.

**How you encountered the issue**

joined valve servers.

**Screenshots**

**Desktop (please complete the following information):**
* Config version: don't say latest, say the actual version, found by running `version_comfig` in console
* Config preset and addon(s): Ultra
* Custom values (if any):
* DX level: 98
* Launch options: -nojoy -novid
* Operating system (OS): Windows
* Graphics card (GPU): GTX 1050 ti
* Graphics driver version: 
* Processor (CPU): I7 
* TF2 on HDD or SSD: SSD
* Memory (RAM) size in MB or GB: 24GB

**Comment by Tiagoquix** [comment]
Add this to your `user/autoexec.cfg`:

```
mod_load_anims_async 0
mod_load_mesh_async 0
mod_load_vcollide_async 0
```

**Comment by AtomicTEM** [comment]
It fixed the issue, but what did it do exactly?

**Comment by AtomicTEM** [comment]
Also i was using version 8.3.2

**Comment by AtomicTEM** [comment]
also you may want to update the troubleshooting guide for this issue, because it says to use the old slow I/O addon.

**Comment by mastercoms** [comment]
Fixed in 8.3.5. Use preload_enable.
